tests/filecheck/backend/rvscf_lowering_labels.mlir

```diff
@@ -0,0 +1,112 @@
+// RUN: xdsl-opt -p lower-riscv-scf-to-labels --split-input-file %s | filecheck %s
+
+builtin.module {
+    "riscv_func.func"() ({
+    ^0(%0 : !riscv.reg<a0>, %1 : !riscv.reg<a1>):
+        %2 = riscv.mv %0 : (!riscv.reg<a0>) -> !riscv.reg<a0>
+        %3 = riscv.mv %1 : (!riscv.reg<a1>) -> !riscv.reg<a1>
+        %4 = riscv.li 1 : () -> !riscv.reg<a2>
+        %5 = riscv.li 0 : () -> !riscv.reg<a3>
+        %6 = riscv.fcvt.s.w %5 : (!riscv.reg<a3>) -> !riscv.freg<fa0>
+        %7 = "riscv_scf.for"(%2, %3, %4, %6) ({
```

Comment:
I feel riscv_scf.for could really benefit from custom formatting, such that it is clear what the lb and upper bound are.

Reply:
Good idea, I filed an issue #1454